addons/sale_pdf_quote_builder/controllers/quotation_document.py

```diff
@@ -0,0 +1,33 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import json
+import logging
+
+from odoo import _
+from odoo.http import Controller, request, route
+
+logger = logging.getLogger(__name__)
+
+
+class QuotationDocumentController(Controller):
+
+    @route(
+        '/sale_pdf_quote_builder/quotation_document/upload',
+        type='http',
+        methods=['POST'],
+        auth='user',
+    )
+    def upload_document(self):
+        files = request.httprequest.files.getlist('ufile')
+        result = {'success': _("All files uploaded")}
+        for ufile in files:
+            try:
+                mimetype = ufile.content_type
+                request.env['quotation.document'].create({
+                    'name': ufile.filename, 'mimetype': mimetype, 'raw': ufile.read()
+                })
+            except Exception as e:
```

Comment:
:point_up: @chevalierv FYI
I'll change it in this version if the mimetype isn't pdf like, but I know you'll have a common for this code and the product document ones. However, if I can ensure it's PDF for the quotation docs, it's not the case for the product doc. Just so you can be aware of this change. But I guess you won't change too much the controllers, so this might not impact you?

Reply:
The upload route didn't change so it should not impact :)